components/rich_text_editor/extensions/suggestion/SuggestionList.vue

```diff
@@ -2,14 +2,15 @@
   <div class="d-popover__dialog">
     <ul
       v-show="items.length"
+      ref="suggestionList"
       class="dt-suggestion-list"
     >
       <dt-list-item
         v-for="(item, index) in items"
         :key="index"
```

Comment:
Don't use index for key here it's actually causing a warning to be thrown in vue. Need to use some string that uniquely identifies the item.

Reply:
Fix was not exactly what I meant. Use contactId for the key instead of index.

Reply:
the problem is that this component is used for contact suggestion, emoji suggestion and in future channel suggestion. So probab;y have to pass a unique index ?

Reply:
btw the warning you talking about is this:
```
[Vue warn]: Avoid app logic that relies on enumerating keys on a component instance. The keys will be empty in production mode to avoid performance overhead.
```

this is because of listening to events on blur/focus/update and this has been happening since before i took this component.

Reply:
Then make it a generic id field that all of these would use

Reply:
The problem with using index, is it changes if the order of the item in the list changes, then things get all kinds of messed up with reactivity and such, so it's best practice to not use it.

Reply:
> this is because of listening to events on blur/focus/update and this has been happening since before i took this component.

All good, not relevant to this PR then.

Reply:
So i used item.id and i changed the emoji implementation to add a atrribute of id. Unfortunately we dont have a unique id for emoji, so i made id = unicode character which will be unique in either way. Now key can just refer item.id.

Reply:
yeah thats fine, id just just to be SOMETHING unique even if it's a unicode char